core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/conditional/IfMissingOrNullExpression.kt

```diff
@@ -0,0 +1,28 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.conditional
+
+import ch.ergon.dope.resolvable.expression.UnaliasedExpression
+import ch.ergon.dope.validtype.ValidType
+
+class IfMissingOrNullExpression<T : ValidType>(
+    firstExpression: UnaliasedExpression<T>,
+    secondExpression: UnaliasedExpression<T>,
+    vararg additionalExpressions: UnaliasedExpression<T>,
+) : ConditionalExpression<T>("IFMISSINGORNULL", firstExpression, secondExpression, *additionalExpressions)
+
+fun <T : ValidType> ifMissingOrNull(
+    firstExpression: UnaliasedExpression<T>,
+    secondExpression: UnaliasedExpression<T>,
+    vararg additionalExpressions: UnaliasedExpression<T>,
+) = IfMissingOrNullExpression(firstExpression, secondExpression, *additionalExpressions)
+
+class CoalesceExpression<T : ValidType>(
```

Comment:
Why do you have 2 Classes in one File?

Reply:
Ah, I just read the documentation, and the Coalesce is just an alias for the IfMissingOrNull. Correct?

Reply:
Yes, correct; I left them in the same file analogous to how I did it with the different `ArraySymmetricDifferenceExpression` classes. But I could extract them into their own file if you wish 🙂